cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
`= { 0 }` was explicitly exempted from the `missing-field-initializers` warning in gcc in 2011: https://gcc.gnu.org/viewcvs/gcc?view=revision&revision=172857